### PR TITLE
Added Slovenian Translation [fixes #544]

### LINKED ISSUE #544: Site Translation: Add Slovenian
### Prize Bounty
100 DAI

### Challenge Description
Slovenian translations are completed, reviewed & ready to to be added to the website!

### Submission Requirements
[This PR](https://github.com/ethereum/ethereum-org-website/pull/282/files) may be a useful reference. Below are the steps required to add a new language:
1. Download the [markdown files from CrowdIn](https://crowdin.com/project/ethereumfoundation/sl#). You may need to sign up for CrowdIn & join our project in order to download the files - here's the [invite link](https://crowdin.com/project/ethereumfoundation/invite).
2. Add markdown files in a new /docs/sl/ directory
3. Add nav to config.js (`locales` key)
4. Update the homepage markdown file (the root index.md) to use the `HomePage.vue` component
  - Move that content into translations.js (again, it helps to follow [this previous PR](https://github.com/ethereum/ethereum-org-website/pull/282/files))
  - Add `layout: home` in page [front matter](https://vuepress.vuejs.org/guide/frontmatter.html)
5. In each newly added markdown file...
- [Front matter](https://vuepress.vuejs.org/guide/frontmatter.html)
  - Add `meta` property (i.e. `og:title`) - check out other translation markdown files for reference
  - Confirm it has (correct) `lang` property
  - Remove `date` fields (if they exist)
  - Add `sidebar: auto` for /learn & /developers pages
- Add [explicit heading IDs](https://www.markdownguide.org/extended-syntax/#heading-ids) to each primary header (`#`) & secondary header (`##`) to match the English file versions. For more context on why we do this, [see here](https://github.com/ethereum/ethereum-org-website/issues/272)
  - e.g. `## Začíname {#getting-started}` from /docs/sk/developers/index.md
- Update relative links (i.e. add language path)
  - e.g. a link to the /use page within the /docs/sl/beginners/index.md file should link to ethereum.org/sl/use, not ethereum.org/use

### Judging Criteria
The prize will be awarded to the first valid submission.

### Winner Announcement Date
We will review work within a week of submission. Payouts will occur within a week of submission acceptance.

### PR DESCRIPTION
**Pull request by karansinghgit**
### Description

Added Slovenian translation. All files changed with respect to the guidelines specified in PR #282 

### Related Issue
Solves Issue #544 

### Screenshots (if appropriate):
None

### CODE REVIEW

File: docs/sl/beginners/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+**Skupnost Ethereuma je največja in najbolj aktivna skupnost veriženja blokov na svetu.** Vključuje razvijalce jedra protokola, raziskovalce kriptoekonomije, cypherpunke, rudarske organizacije, imetnike ETH, razvijalce aplikacij, navadne uporabnike, anarhiste, podjetja na seznamu Fortune 500 in od zdaj naprej tudi **vas**.
+
+**Ethereum ni pod nadzorom nobenega podjetja ali centralizirane organizacije.** Ethereum vzdržuje in izboljšuje raznolika globalna skupnost, ki s svojim delom prispeva k vsemu, od jedra protokola do aplikacij za uporabnike. To spletno mesto je tako kot celoten projekt Ethereum postavila skupina ljudi, ki delajo za isti cilj.
+
+**Pozdravljeni v Ethereumu.**
+
+**Ne veste, kam naprej?**
+
+- Želite začeti uporabljati Ethereum? [ethereum.org/sl/use](/use/)
```
Comment (samajammin) [inline docs/sl/beginners/index.md]: These links should be updated to the equivalent Slovenian pages (i.e. /use/ --> /sl/use/)

File: docs/sl/learn/index.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  - property: og:title
+    content: Učenje | Ethereum
+lang: sl
+sidebar: auto
+sidebarDepth: 0
+---
+
+# Spoznajte Ethereum {#learn-about-ethereum}
+
+**Pozdravljeni v [ethereum.org/sl/learn](/learn/) – to je zbirka virov, ki vam bodo v pomoč pri spoznavanju Ethereuma.**Na tej strani so tehnični **in** netehnični članki, vodniki in viri. Če ste ravnokar odkrili Ethereum, [priporočamo, da začnete tukaj](/beginners/).
```
Comment (samajammin) [inline docs/sl/learn/index.md]: Same here (/learn/ --> /sl/learn) & (/beginners/ --> /sl/beginners)

File: docs/sl/learn/index.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+- [How does Ethereum work, anyway?](https://medium.com/@preethikasireddy/how-does-ethereum-work-anyway-22d1df506369) *Sept 27, 2017 - Preethi Kasireddy*
+- [A Gentle Introduction to Ethereum](https://bitsonblocks.net/2016/10/02/gentle-introduction-ethereum/) *Oct 2, 2016 - Antony Lewis*
+- [Introduction to Blockchain through Cryptoeconomics - Part 1](https://blockchainatberkeley.blog/introduction-to-blockchain-through-cryptoeconomics-part-1-bitcoin-369f245067f9) *Jan 26, 2018 - Zubin Koticha*
+- [Introduction to Blockchain through Cryptoeconomics - Part 2](https://medium.com/mechanism-labs/introduction-to-bitcoin-through-cryptoeconomics-part-2-proof-of-work-and-nakamoto-consensus-1252f6a6c012) *19. julij 2018 - Zubin Koticha*
+
+## Pametne pogodbe {#smart-contracts}
+
+Preprosto povedano je "pametna pogodba" samo delček kode, ki se izvaja v Ethereumu. Imenujemo jo "pogodba", ker lahko koda, se izvaja v Ethereumu, nadzoruje stvari, ki imajo vrednost, kot so ETH in druga digitalna sredstva.
+
+- Ali se želite naučiti, kako programirati pametne pogodbe v Ethereumu? [ethereum.org/sl/developers](/developers/)
```
Comment (samajammin) [inline docs/sl/learn/index.md]: Same here.

File: docs/sl/learn/index.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+- [What does it mean to mine Ethereum?](https://docs.ethhub.io/using-ethereum/mining/) *Pogosto posodobljeno – Ethhub*
+- [How Ethereum Mining Works](https://www.coindesk.com/information/ethereum-mining-works) *Coindesk*
+
+V ETH 2.0 bo Ethereum prešel na drugačen sistem, imenovan "dokaz deleža" (“Proof of Stake”). [Več o ETH 2.0 preberite spodaj](./#eth-2-0).
+
+## Odjemalci in vozlišča {#clients-and-nodes}
+
+Omrežje Ethereum je sestavljeno iz številnih vozlišč in v vsakem od njih se izvaja združljiva odjemalska programska oprema. Večina vozlišč uporablja enega od dveh odjemalcev: [Geth](https://geth.ethereum.org/) (napisan v programskem jeziku Go) in [Parity](https://www.parity.io/ethereum/) (napisan v programskem jeziku Rust).
+
+- Ali želite izvajati svoje vozlišče? → [ethereum.org/developers](/developers/#clients-running-your-own-node)
```
Comment (samajammin) [inline docs/sl/learn/index.md]: Same here.

**Comment by samajammin** [comment]
@all-contributors please add @karansinghgit for code

**Comment by samajammin** [comment]
Hey @karansinghgit - please head here to fulfill the work: https://explorer.bounties.network/bounty/3752

Once that's complete, we'll get this bounty paid out to you! Thanks again.

**Comment by karansinghgit** [comment]
@samajammin Filled the form! Thanks!

**Comment by samajammin** [comment]
Hey @karansinghgit can you please confirm your wallet address? We have a submission on the Bounties Network but it’s empty so I want to make sure we have the right person.

Feel free to ping me on Telegram (@samajammin) if you prefer to share it there.

Thanks.

**Pull request merged**